frontend/src/v4/routes/components/textField/textField.styles.ts

```diff
@@ -124,3 +124,11 @@ export const CopyButton = styled(ContainedButton)`
 		margin-left: 8px;
 	}
 `;
+
+export const ExpandAction = styled.span<{ top?: boolean }>`
```

Comment:
Is there reason for having this same ExpandAction component in _textField.styles_ and in _riskDetails.styles_?

Reply:
It was done in order get rid of a cyclical dependency. The code above will be kept whereas the namesake in `riskDetails.styles` will be deleted